src/plugins/animatedBackgrounds/index.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2022 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { Devs } from "@utils/constants";
+import { OptionType } from "@utils/types";
+import definePlugin from "@utils/types";
+import './styles.css';
+
+export default definePlugin({
+    name: "AnimatedBackgrounds",
+    description: "Allows you to use a YouTube video URL as an animated background.",
+    authors: [Devs.soul_fire_],
+    patches: [],
+    options: {
+        source: {
+            description: "Source URL to replace the background (a background-supporting theme must be installed first)",
```

Comment:
```suggestion
            description: "Source URL to replace the background",
```

I would use a settings about component to say this, and provide an basic example theme